src/pages/[platform]/build-a-backend/auth/connect-your-frontend/sign-in/index.mdx

```diff
@@ -830,7 +830,7 @@ signInWithRedirect({ provider: {
 }})
 ```
 
-### Auto sign-in
+## Auto sign-in
```

Comment:
I believe this deserves it's own section, previously was nested beneath the [Sign in with an external identity provider](https://docs.amplify.aws/react/build-a-backend/auth/connect-your-frontend/sign-in/#sign-in-with-an-external-identity-provider) section.